### ISSUE TITLE
Use CF-compliant `standard_name` is internal nomenclature instead of ERA5

### ISSUE THREAD

**Issue by leifdenby** [EUREC4A-UK/lagtraj#179]
To ease the use of input data from other sources that ERA5 I think we should switch to accessing variables using the CF-compliant standard name instead. This is made easy using the [cf-xarray](https://cf-xarray.readthedocs.io/en/latest/examples/introduction.html#feature-accessing-variables-by-standard-names) package. We could then also define the set of minimum required variables using the CF-compliant standard names too and generalise the variables we use in forcings, i.e. in https://github.com/EUREC4A-UK/lagtraj/blob/master/lagtraj/forcings/profile_calculation.py#L12

what do you think @sjboeing ?

**Comment by sjboeing** [comment]
@leifdenby: yes, this sounds good, though I will need to think it through: it would involve adding a dictionary for the mapping for each input model, initially I had thought of using the ERA5 names as a reference for the other models. IFS naming is likely to be very similar to ERA5.

**Comment by leifdenby** [comment]
> @leifdenby: yes, this sounds good, though I will need to think it through: it would involve adding a dictionary for the mapping for each input model, initially I had thought of using the ERA5 names as a reference for the other models. IFS naming is likely to be very similar to ERA5.

Yes, I think it will mean that when targeting a specific output model we would refer to the variables to use by their standard name.
